cmd/generate/Dockerfile.template

```diff
@@ -21,6 +21,11 @@ USER 65532
 
 COPY --from=builder /usr/bin/main /usr/bin/main
 
+{{- if .sym_links }}
+
+CMD {{ .sym_links }}
+{{- end }}
```

Comment:
func-utils is using `RUN`, I would also just call this `.postBuildInstructions` and remove the CMD or RUN

```
RUN ln -s /usr/local/bin/func-util /usr/local/bin/deploy && \
    ln -s /usr/local/bin/func-util /usr/local/bin/scaffold && \
    ln -s /usr/local/bin/func-util /usr/local/bin/s2i
```

{{ .postBuildInstructions }}

Reply:
Good point about CMD vs RUN

Reply:
Also switched to the postbuildinstructions